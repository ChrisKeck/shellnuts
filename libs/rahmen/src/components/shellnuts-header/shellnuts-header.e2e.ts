import { test,expect } from '@playwright/test';

test.describe('shellnuts-header', async() => {
  test.beforeEach(async({page})=>{
    await page.goto("http://localhost:3334")
  })
  test('renders', async ({page}) => {
    await expect(page).toHaveURL("http://localhost:3334/")
  });
});
