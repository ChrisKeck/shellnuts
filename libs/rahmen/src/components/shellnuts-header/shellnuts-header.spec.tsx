import { newSpecPage } from '@stencil/core/testing';
import { ShellnutsHeader } from './shellnuts-header';

describe('shellnuts-header', () => {
  it('renders', async () => {
    const { root } = await newSpecPage({
      components: [ShellnutsHeader],
      html: '<shellnuts-header></shellnuts-header>',
    });
    expect(root).toEqualHtml(`
      <shellnuts-header>
        <mock:shadow-root>
          <div>
            Hello, World! I'm
          </div>
        </mock:shadow-root>
      </shellnuts-header>
    `);
  });

  it('renders with values', async () => {
    const { root } = await newSpecPage({
      components: [ShellnutsHeader],
      html: `<shellnuts-header first="Stencil" last="'Don't call me a framework' JS"></shellnuts-header>`,
    });
    expect(root).toMatchSnapshot()
  });
});
