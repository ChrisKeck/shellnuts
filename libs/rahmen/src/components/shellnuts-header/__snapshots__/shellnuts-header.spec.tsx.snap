// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`shellnuts-header renders with values 1`] = `
<shellnuts-header first="Stencil" last="'Don't call me a framework' JS">
  <mock:shadow-root>
    <div>
      Hello, World! I'm Stencil undefined 'Don't call me a framework' JS
    </div>
  </mock:shadow-root>
</shellnuts-header>
`;
